the-collab-lab/tcl-70-smart-shopping-list
# Updates instruction for home and list page

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓    | :memo: New feature |

## Acceptance Criteria

- [x] Instructions for interacting with items on the home page have been updated to improve clarity.
- [x] New instructions for marking items as purchased and understanding the 24-hour repurchase restriction have been added to the list page.

## Description

This PR focuses on improving user guidance within the application by updating existing instructions on the home page and introducing new instructions on the list page. 

### Updates to Home Page Instructions

We've clarified the steps a user should follow when interacting with the home page, ensuring they have a clear understanding of how to navigate through the app right from the start.

### New Instructions for the List Page

On the list page, we've added crucial instructions that inform users about the functionality of marking items as purchased. This includes an explanation that once an item is marked as purchased, it cannot be marked again within a 24-hour period to simulate realistic shopping behavior and prevent accidental repurchases.

## Description
### Before
<img width="1393" alt="image" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/121919044/a9d926b6-c39a-43f1-b811-37507e8bf905">
<img width="1393" alt="image" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/121919044/3d515c8d-e05f-4d1e-a0a6-9b8008b61bc4">

### After
<img width="1419" alt="image" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/121919044/867e667d-d5e0-4d77-b441-749dbe1ae8eb">
<img width="1393" alt="image" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/121919044/eb588d86-5567-4dfc-b777-b79b09d74474">

## Testing Steps / QA Criteria

* From your terminal, switch to this branch with `git checkout ag-modify-instruction`.
* Run `npm start` to launch the app.
* Verify that the updated instructions are present on the home page and that the new instructions are correctly displayed on the list page.
* Interact with the list items to ensure that the 24-hour repurchase restriction behaves as expected.
